docs/rules/camelcase.md

```diff
@@ -217,6 +219,28 @@ Examples of **correct** code for this rule with the `{ "ignoreImports": true }`
 import { snake_cased } from 'mod';
 ```
 
+### ignoreGlobals: false
+
+Examples of **incorrect** code for this rule with the default `{ "ignoreGlobals": false }` option:
+
+```js
+/*eslint camelcase: ["error", {ignoreGlobals: false}]*/
+/* global some_property */
+
+const property = some_property;
```

Comment:
Can we change the name, `"property"` (and `"some_property"`) might be confusing in this context?

maybe `const foo = no_camelcased;`

(the same applies to examples of correct code below)